src/components/CookiesBanner/index.tsx

```diff
@@ -263,16 +264,20 @@ const CookiesBanner = isDesktop
         }
 
         const { acceptedNecessary, acceptedSupportAndUpdates, acceptedAnalytics } = cookiesState
+
         setLocalNecessary(acceptedNecessary)
         setLocalSupportAndUpdates(acceptedSupportAndUpdates)
         setLocalAnalytics(acceptedAnalytics)
       }, [setLocalNecessary, setLocalSupportAndUpdates, setLocalAnalytics, openBanner])
 
-      // Load or unload analytics depending on user choice
       useEffect(() => {
         localAnalytics ? loadGoogleAnalytics() : unloadGoogleAnalytics()
       }, [localAnalytics])
 
+      useEffect(() => {
+        localSupportAndUpdates ? loadBeamer() : window.Beamer && unloadBeamer()
```

Comment:
I would suggest using if statements here instead. It's somewhat hard to follow.

Reply:
I would rather move the additional `window.Beamer` into `unloadBeamer` so it is consistent with `unloadGoogleAnalytics`

Reply:
Could you also move it back to where it was? Just to minimize the fix.